docs/_includes/_google-tag-manager.html

```diff
@@ -1,5 +1,5 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-18433785-19"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>
```

Comment:
Where is this `site.google_analytics` defined?

Reply:
Here: https://github.com/bow-swift/bow/blob/99f7f7506011be06da535b1980cebec8ccd31503/docs/_config.yml#L21

Reply:
In Jekyll the way to read values declared in the config file is that way. We could also store it in a different `yml` file though.